float32 start_deg
float32 start_dist
float32 mid_deg
float32 mid_dist
float32 end_deg      # deg == degree
float32 end_dist     # dist == distance 